dev-python/terminado/terminado-0.8.1.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6} )
+
+inherit distutils-r1
+
+DESCRIPTION="Terminals served to term.js using Tornado websockets"
+HOMEPAGE="https://pypi.python.org/pypi/terminado https://github.com/jupyter/terminado"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+SLOT="0"
+LICENSE="BSD-2"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="test"
+
+RDEPEND="
+	dev-python/ptyprocess[${PYTHON_USEDEP}]
+	>=www-servers/tornado-0.4[${PYTHON_USEDEP}]"
+DEPEND="${RDEPEND}
+	test? ( dev-python/nose[${PYTHON_USEDEP}] )"
```

Comment:
FWICS, upstream doesn't require nose at all. They list it in README but they use py.test on travis. The test file seems to use pure `unittest` only, so please remove the unnecessary dep and run the test directly.

Reply:
Yes you're right, the project's README.md is actually misleading. Fixed it.